posts/2020-09-01-jakarta-microprofile-beta-200010.adoc

```diff
@@ -0,0 +1,355 @@
+---
+layout: post
+title: "New features for MicroProfile 4.0 and more Jakarta EE 9 features in Open Liberty 20.0.0.10 beta"
+categories: blog
+author_picture: https://avatars3.githubusercontent.com/jakub-pomykala
+author_github: https://github.com/jakub-pomykala
+seo-title: New features for MicroProfile 4.0 and more Jakarta EE 9 features in Open Liberty 20.0.0.10 beta - OpenLiberty.io
+seo-description: Open Liberty 20.0.0.10 beta comes with new features for MicroProfile 4.0 and more Jakarta EE 9 features. Give it a try by adding either Jakarta EE 9 Beta features package coordinates or the All Beta features package coordinates to your build tools.
+blog_description: Open Liberty 20.0.0.10 beta comes with new features for MicroProfile 4.0 and more Jakarta EE 9 features. Give it a try by adding either Jakarta EE 9 Beta features package coordinates or the All Beta features package coordinates to your build tools.
+---
+= New features for MicroProfile 4.0 and more Jakarta EE 9 features in Open Liberty 20.0.0.10 beta
+Jakub Pomykala <https://github.com/jakub-pomykala>
+:imagesdir: /
+:url-prefix:
+:url-about: /about/
+
+In this release of Open Liberty Beta you can try the new features for MicroProfile 4.0 which are Fault Tolerance 3.0, Matric 3.0, Health 3.0 and OpenTracing 2.0 plus more Jakarta EE 9 features.
+
+We have two beta packages for link:{url-about}[Open Liberty]:
+
+* <<allbeta, All Beta Features>>: a new, larger package that contains all Open Liberty beta features (including Jakarta EE 9 beta features) and GA features and functions.
+* <<jakarta, Jakarta EE 9 Beta Features>>: a lightweight package that contains only the Jakarta EE 9 features.
+
+This means that you can now try out our in-development Open Liberty features by just adding the relevant coordinates to your build tools (either the All Beta Features package or the Jakarta EE 9 Beta Features package, depending on what you want to try).
+
+If you try either package, <<feedback, let us know what you think>>.
+
+[#allbeta]
+== All Beta Features package
+
+The All Beta Features package includes the following beta features:
+
+* <<fault, MicroProfile Fault Tolerance 3.0>>
+* <<metric, MicroProfile Metric 3.0>>
+* <<health, MicroProfile Health 3.0>>
+* <<opentracing, MicroProfile OpenTracing 2.0>>
+* <<jakarta, Jakarta EE 9 Beta Features package>>
+
+
+[#fault]
+=== MicroProfile Fault Tolerance 3.0
+
+MicroProfile Fault Tolerance allows you to easily apply strategies for mitigating failure to their code. It provides annotations which you can add to methods to use bulkhead, circuit breaker, retry, timeout and fallback strategies.
+
+Fault Tolerance 3.0 overhauls the metrics automatically exported by Fault Tolerance to take advantage of metric tags to make it easier to use those metrics to understand when and where your application is failing.
```

Comment:
This sentence has 3 instances of 'metrics'.  How about.
Fault Tolerance 3.0 overhauls the metrics that are automatically exported by Fault Tolerance to take advantage of tags and make it easier to use those metrics to understand when and where your application is failing.

Reply:
Done